ui/src/app/kvm/views/KVMDetails/LxdProject/LxdProject.tsx

```diff
@@ -0,0 +1,35 @@
+import { Spinner } from "@canonical/react-components";
+import { useSelector } from "react-redux";
+import { Redirect } from "react-router";
+
+import { useWindowTitle } from "app/base/hooks";
+import podSelectors from "app/store/pod/selectors";
+import type { Pod } from "app/store/pod/types";
+import { PodType } from "app/store/pod/types";
+import type { RootState } from "app/store/root/types";
+
+type Props = { id: Pod["id"] };
+
+const LxdProjects = ({ id }: Props): JSX.Element => {
+  const pod = useSelector((state: RootState) =>
+    podSelectors.getById(state, Number(id))
+  );
+
+  useWindowTitle(`LXD project ${pod?.project || ""}`);
+
+  if (!pod) {
```

Comment:
I think this should check for loading here and add another condition for visiting a pod that doesn't exists (a "this pod does not exist" message or something).

Reply:
Good call! Updating now